index.html

```diff
@@ -41,12 +41,114 @@ <h1>Emmanuel Mensah</h1>
+                        </ul>
+                        <div class="project_click_class text_body_font">
+                            <p> See this project </p>
+                            <div class="arrow_key"></div>
+                         </div>
+                    </div>
+
+                    <div class="grid_3 grid_picture"></div>
+                    <div class="grid_row">
+                        <p class="pro_text text_body_font">Tonic</p>
+                        <ul class="list_category text_body_font">
+                            <li >Html</li>
+                            <li >CSS</li>
+                            <li >Javascript</li>
+                        </ul>
+                        <div class="project_click_class text_body_font">
+                            <p> See this project </p>
+                            <div class="arrow_key"></div>
+                         </div>
+                    </div>
+
+                    <div class="grid_4 grid_picture"></div>
+                    <div class="grid_row">
+                        <p class="pro_text text_body_font">Multi-Post Stories</p>
+                        <ul class="list_category text_body_font">
+                            <li>Html</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                        <div class="project_click_class text_body_font">
+                            <p> See this project </p>
+                            <div class="arrow_key"></div>
+                         </div>
+                    </div>
+
+                    <div class="grid_5 grid_picture"></div>
+                    <div class="grid_row">
+                        <p class="pro_text text_body_font">Tonic</p>
+                        <ul class="list_category text_body_font">
+                            <li>Html</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                        <div class="project_click_class text_body_font">
+                            <p> See this project </p>
+                            <div class="arrow_key"></div>
+                         </div>
+                    </div>
+
+                    <div class="grid_6 grid_picture"></div>
+                    <div class="grid_row">
+                        <p class="pro_text text_body_font">Multi-Post Stories</p>
+                        <ul class="list_category text_body_font">
+                            <li>Html</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                        <div class="project_click_class text_body_font">
+                            <p> See this project </p>
+                            <div class="arrow_key"></div>
+                         </div>
+                    </div>
+
+
+                </div>
+            </section>

```

Comment:
- [ ] Please go on and include the about section as it is a requirement in this milestone 🙏🏻.

Reply:
Thanks, @Cathella, it will be done.

Reply:
@Cathella https://github.com/principles31/portfolio/pull/3 CHeck this pull request with the about page added.